btcclient/query.go

```diff
@@ -66,13 +66,16 @@ func (c *Client) getBestBlockHashWithRetry() (*chainhash.Hash, error) {
 		err       error
 	)
 
-	if err := retry.Do(c.retrySleepTime, c.maxRetrySleepTime, func() error {
+	if err := retry.Do(func() error {
 		blockHash, err = c.GetBestBlockHash()
 		if err != nil {
 			return err
 		}
 		return nil
-	}); err != nil {
+	},
+		retry.Delay(c.retrySleepTime),
```

Comment:
are those those options ever set in `btclient` ?

Reply:
Good catch, fixed in [f7c96e3](https://github.com/babylonlabs-io/vigilante/pull/29/commits/f7c96e39b7bc9ae631eee54b16d44d2e889fba60)